tofu/config/demo.efilerapi.org/main.tf

```diff
@@ -14,99 +14,12 @@ module "backend" {
   environment = "demo"
 }
 
-module "logging" {
-  source = "github.com/codeforamerica/tofu-modules-aws-logging?ref=2.1.0"
-
-  project     = "efiler-api"
-  environment = "demo"
-}
-
-module "secrets" {
-  source = "github.com/codeforamerica/tofu-modules-aws-secrets?ref=1.0.0"
-
-  project     = "efiler-api"
-  environment = "demo"
-
-  secrets = {
-  }
-}
-
-module "vpc" {
-  source = "github.com/codeforamerica/tofu-modules-aws-vpc?ref=1.1.1"
-
-  project        = "efiler-api"
-  environment    = "demo"
-  cidr           = "10.0.48.0/22"
-  logging_key_id = module.logging.kms_key_arn
+module "efiler_api" {
+  source = "../../modules/efiler_api"
 
+  environment     = "demo"
+  domain          = "demo.efiler-api.fileyourstatetaxes.org"
```

Comment:
could this string be interpolated in the module like `"${environment}.efiler-api.fileyourstatetaxes.org"`?